dags/pytorch_xla/pytorchxla_torchvision.py

```diff
@@ -72,3 +72,25 @@
       US_CENTRAL1,
       "gpu-uc1",
   ).run()
+
+  resnet_v100_2x2_spmd = task.GpuGkeTask(
+      test_config.JSonnetGpuTest.from_pytorch("pt-nightly-resnet50-spmd-batch-fake-v100-x2x2"),
+      US_CENTRAL1,
+      "gpu-uc1",
+  ).run()
+
+  resnet_v100_2x2 >> resnet_v100_2x2_spmd
```

Comment:
I wonder what this line does.

Reply:
This is a control dependency between these two tests. It requires that the MP version of the test passes before attempting the SPMD version.